src/main.js

```diff
@@ -24,6 +24,7 @@ Vue.use(VueProgressBar, {
 		opacity: '0s',
 		termination: 300,
```

Comment:
@mcstover termination seems to be how many ms the progress bar stays on the screen after reaching 100%